receiver/jaegerreceiver/factory.go

```diff
@@ -38,11 +38,10 @@ const (
 	protoThriftCompact = "thrift_compact"
 
 	// Default endpoints to bind to.
-	defaultGRPCBindEndpoint                = "0.0.0.0:14250"
-	defaultHTTPBindEndpoint                = "0.0.0.0:14268"
-	defaultThriftCompactBindEndpoint       = "0.0.0.0:6831"
-	defaultThriftBinaryBindEndpoint        = "0.0.0.0:6832"
-	defaultAgentRemoteSamplingHTTPEndpoint = "0.0.0.0:5778"
```

Comment:
Sorry, I should have been clearer on the original issue: we should remove this, but we need to first deprecate for at least a couple of versions. So, at this point, we need to only detect whether this is being used and log a info/warn-level message, stating that this port will disappear by version X (0.60.0 is the earliest at this point, I believe).

Reply:
i am addressing this in #13408 and closing the pr until we finally remove the endpoint.